azurerm/internal/services/appplatform/tests/resource_arm_spring_cloud_test.go

```diff
@@ -0,0 +1,187 @@
+					resource.TestCheckResourceAttrSet(data.ResourceName, "service_id"),
+					resource.TestCheckResourceAttrSet(data.ResourceName, "version"),
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.%", "2"),
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.env", "test"),
+					resource.TestCheckResourceAttr(data.ResourceName, "tags.version", "1"),
+				),
+			},
+			data.ImportStep(),
+		},
+	})
+}
+
+func testCheckAzureRMSpringCloudExists(resourceName string) resource.TestCheckFunc {
+	return func(s *terraform.State) error {
+		rs, ok := s.RootModule().Resources[resourceName]
+		if !ok {
+			return fmt.Errorf("Spring Cloud not found: %s", resourceName)
+		}
+
+		name := rs.Primary.Attributes["name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+
+		client := acceptance.AzureProvider.Meta().(*clients.Client).AppPlatform.ServicesClient
+		ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+		if resp, err := client.Get(ctx, resourceGroup, name); err != nil {
+			if utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Bad: Spring Cloud Service %q (Resource Group %q) does not exist", name, resourceGroup)
+			}
+			return fmt.Errorf("Bad: Get on AppPlatform.ServicesClient: %+v", err)
+		}
+
+		return nil
+	}
+}
+
+func testCheckAzureRMSpringCloudDestroy(s *terraform.State) error {
+	client := acceptance.AzureProvider.Meta().(*clients.Client).AppPlatform.ServicesClient
+	ctx := acceptance.AzureProvider.Meta().(*clients.Client).StopContext
+
+	for _, rs := range s.RootModule().Resources {
+		if rs.Type != "azurerm_spring_cloud" {
+			continue
+		}
+
+		name := rs.Primary.Attributes["name"]
+		resourceGroup := rs.Primary.Attributes["resource_group_name"]
+
+		if resp, err := client.Get(ctx, resourceGroup, name); err != nil {
+			if !utils.ResponseWasNotFound(resp.Response) {
+				return fmt.Errorf("Bad: Get on AppPlatform.ServicesClient: %+v", err)
+			}
+		}
+
+		return nil
+	}
+
+	return nil
+}
+
+func testAccAzureRMSpringCloud_basic(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_spring_cloud" "test" {
+  name                     = "acctestsc-%d"
+  location                 = azurerm_resource_group.test.location
+  resource_group_name      = azurerm_resource_group.test.name
+
+  tags = {
+    env = "test"
+  }
+}
+
+`, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
+}
+
+func testAccAzureRMSpringCloud_complete(data acceptance.TestData) string {
+	return fmt.Sprintf(`
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_spring_cloud" "test" {
+  name                     = "acctestsc-%d"
+  location                 = azurerm_resource_group.test.location
+  resource_group_name      = azurerm_resource_group.test.name
+
+  tags = {
+    env = "test"
```

Comment:
Could we use capitals
```suggestion
    Env = "Test"
```